src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/DatasetCatalogue.kt

```diff
@@ -16,27 +16,43 @@ class DatasetCatalogue {
     @Deprecated(message = "Use largeJiraSeven() instead")
     fun largeJira() = largeJiraSeven()
 
-    fun largeJiraSeven(): Dataset = custom(
-        location = StorageLocation(
-            URI("s3://jpt-custom-datasets-storage-a008820-datasetbucket-1sjxdtrv5hdhj/")
-                .resolve("dataset-d4684761-116b-49ae-9cce-e45cecdcae2a"),
-            Regions.EU_WEST_1
-        ),
-        label = "2M issues, format 7",
-        databaseDownload = ofMinutes(17),
-        jiraHomeDownload = ofMinutes(21)
-    )
+    fun largeJiraSeven(): Dataset {
+        val bucketUri = URI("https://s3-eu-west-1.amazonaws.com/jpt-custom-datasets-storage-a008820-datasetbucket-1sjxdtrv5hdhj/")
+        return Dataset(
+            label = "2M issues, format 7",
+            database = MySqlDatabase(
+                HttpDatasetPackage(
+                    uri = bucketUri.resolve("dataset-d4684761-116b-49ae-9cce-e45cecdcae2a/database.tar.bz2"),
+                    downloadTimeout = ofMinutes(17)
```

Comment:
Should we decrease the timeout here since it is faster now?

Reply:
1. I need to release infrastructure:4.10.0 to test how much we can cut. 
2. I'm not sure how we'll gain on a low-end ec2 machines with gp2. I would keep the timeouts.

Reply:
FYI there is already a PR that is about releasing 4.7.0.